config/sexual-health-services/getSettings.js

```diff
@@ -0,0 +1,11 @@
+const mapping = require('./mapping');
+
+function getSettings() {
+  return {
+    type: 'sexual-health-service',
+    idKey: 'id',
+    mapping
```

Comment:
Changing to the request library is a good as it will follow redirects which is worth having.

The README is a funny one, as the env vars relate to a single docker instance, there aren't any notes on the way env vars are passed and renamed in the docker-compose when deployed or run locally.

Should this be another section in the README.md, something like 'deployed configuration'?

Reply:
I think its worth adding something to the `README`, it doesn't need to be much. When looking at the changes it is pretty obvious what is going on but to have something documented acknowledging it would be helpful for future reference.